packages/build/package.json

```diff
@@ -121,6 +122,7 @@
     "@netlify/nock-udp": "^1.0.0",
     "atob": "^2.1.2",
     "ava": "^4.0.0",
+    "c8": "^7.12.0",
```

Comment:
Do we really need to install ava and its friends in every package?

Reply:
That's the thing in a monorepo `npx lerna add -D ava` will add it into each package but hoist it up to the root. That's best practice and should be done like that.

The rule is every binary or script that is executed in a package should care about the dependencies.

only prettier, husky and eslint are installed on root (as they are run from the root via lint staged)

Tests are run from a package level therefore it's added there.